### ISSUE TITLE
"raw_reg" accessor returns null bytes occasionally

### ISSUE THREAD

**Issue by CR-OmerYampel** [Velocidex/velociraptor#640]
Running the same VQL (below) for the same registry path leads to two different results, the second time, the registry data is just null bytes, even though it is the correct length:

1st run:
![image](https://user-images.githubusercontent.com/57952597/93271781-11859e00-f782-11ea-9066-3e93e212568a.png)
2nd run:
![image](https://user-images.githubusercontent.com/57952597/93271791-164a5200-f782-11ea-8cd8-3c089181ae17.png)

VQL is essentially manipulating the output of this:

```
SELECT
          *
        FROM glob(
          globs=url(scheme="ntfs", path=HivePath, fragment="Software/Microsoft/Windows/CurrentVersion/Explorer/RecentDocs/*").String,
          accessor="raw_reg"
      )
```

**Comment by scudette** [comment]
Can you please confirm what version of client you have? Could it be the same as this:
https://github.com/Velocidex/velociraptor/commit/bc78ec14dc1d0098afdd57524377cb1bbd5a578e#diff-d57663609ca7a0048b8f76b78aa598ef

**Comment by CR-OmerYampel** [comment]
```
version:
  name: velociraptor
  version: 0.4.9
  commit: 6a55926
  build_time: "2020-08-31T10:42:16+10:00"
```

Is the client its happening on

**Comment by CR-OmerYampel** [comment]
This was fixed in 0.5.0 👍
